.vscode/settings.json

```diff
@@ -1,28 +1,21 @@
 {
-  "editor.formatOnPaste": true,
-  "editor.formatOnSave": true,
-  "editor.codeActionsOnSave": {
-    "source.fixAll": true
-  },
-  "vetur.format.defaultFormatter.html": "prettyhtml",
-  "vetur.format.defaultFormatter.js": "prettier-eslint",
-  "workbench.colorCustomizations": {
-    "activityBar.activeBackground": "#785ecf",
-    "activityBar.activeBorder": "#da9884",
-    "activityBar.background": "#785ecf",
-    "activityBar.foreground": "#e7e7e7",
-    "activityBar.inactiveForeground": "#e7e7e799",
-    "activityBarBadge.background": "#da9884",
-    "activityBarBadge.foreground": "#15202b",
-    "statusBar.background": "#5839c1",
-    "statusBar.border": "#5839c1",
-    "statusBar.foreground": "#e7e7e7",
-    "statusBarItem.hoverBackground": "#785ecf",
-    "titleBar.activeBackground": "#5839c1",
-    "titleBar.activeForeground": "#e7e7e7",
-    "titleBar.border": "#5839c1",
-    "titleBar.inactiveBackground": "#5839c199",
-    "titleBar.inactiveForeground": "#e7e7e799"
-  },
-  "peacock.color": "#5839c1"
-}
+    "workbench.colorCustomizations": {
```

Comment:
You might have to add this after the commit where you put the the preferences into gitignore

Reply:
I don't know what went wrong. It's still in dev-branch. Let me remove that again.